member.go

```diff
@@ -70,6 +70,17 @@ func (c *Client) Member(nick string) (member *Member, err error) {
 	return
 }
 
+func (c *Client) MemberFromToken(token string) (member *Member, err error) {
```

Comment:
In my use case I get users' Trello tokens and want to fetch basic information from their profiles, so I must call `/1/tokens/<user-token>/member`. I could instantiate another `Client` like in your example, it is complicated and -- I imagine, unnecessarily heavy, but it will work.

Anyway, are you against having a lot of methods mapping exactly from all the Trello API endpoints?